dev/fattest.simplicity/src/componenttest/topology/impl/LibertyServer.java

```diff
@@ -1105,11 +1114,40 @@ public void swapInServerXMLFromPublish(String fileName) throws Exception {
         publishServerXML.copyToDest(serverXML, false, true);
     }
 
+    /*
+     * This method and its getter will change the validate apps setting for all servers, even ones running on other tests
+     * use setValidateAppsLocal to configure the setting for just the current server.
+     */
+    @Deprecated
     public static void setValidateApps(boolean validateApp) {
-        validateApps = validateApp;
+        validateAppsGlobal = Optional.of(validateApp);
     }
 
+    //This gets the value of validateAppsGlobal, or if nothing was explicitly set it returns what would have been the value
+    //under the previous code where validateApps was a static variable shared across all servers
+    @Deprecated
     public boolean getValidateApps() {
+        //orElse(false) because even if you change DEFAULT_VALIDATE_APPS that would be after
+        //validateAppsGlobal went through its static assignment, which used to be to the static
+        //value of DEFAULT_VALIDATE_APPS; which is true
+        return validateAppsGlobal.orElse(true);
+    }
```

Comment:
`DEFAULT_VALIDATE_APPS` is final, so it's safe to use it here and assume it hasn't changed.